tests/ro/creator/data/katoomba-rainfall/example_blueprint.yml

```
@@ -0,0 +1,59 @@
+    - doi:10.100.10010/something
+  name: My first Research Object Crate
+  description: >
+    This is a simple example of a Research Object Crate
+    that includes a few files and some metadata.
+
+    It is intended to test a new ro-creator component
+  datePublished: 2024-02-03
+  license: https://creativecommons.org/licenses/by/4.0/
+
+  agents:
+    https://orcid.org/0000-0002-9648-6484: # Marc Portier
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)